packages/sync/src/class-actions.php

```diff
@@ -503,9 +504,18 @@ public static function do_cron_sync_by_type( $type ) {
 					sleep( $delay );
 				}
 			}
+			$executions ++;
+
+			// Explicitly only allow 1 do_full_sync call until issue with Immediate Full Sync is resolved.
+			// For more context see p1HpG7-9pe-p2.
+			if ( 'full_sync' === $type && $executions > 1 ) {
```

Comment:
Can we remove the `$executions` count and just move the break under `$result = ...` for the `full_sync` case? This should be equivalent.

Reply:
I was thinking executions would be a good case for future changes. This patch is to enforce the current behavior as seen in 8.3+. However pre 8.3 we did allow multiple sends since each send waited for the WP.com response. 

by not having executions included we'd another method of testing iterations during the cron hook.